<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;STKaiti&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_granjie">
         <property name="text">
          <string>官阶</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie">
         <property name="text">
          <string>从九品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_name">
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>宋明</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_granzhi">
         <property name="text">
          <string>官职</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granzhi">
         <property name="text">
          <string>司务</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_five">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>100</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 66), stop:0.65 rgba(159, 235, 148, 128), stop:0.721925 rgba(255, 238, 150, 255), stop:0.77 rgba(255, 128, 128, 255), stop:0.89 rgba(191, 128, 255, 128), stop:1 rgba(0, 0, 0, 0));
font: 20pt &quot;STKaiti&quot;;</string>
    </property>
    <property name="text">
     <string>!五连抽!</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_suishu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/oldpaper.png);
font: 18pt &quot;STKaiti&quot;;</string>
    </property>
    <property name="text">
     <string>岁数：18</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="Label_gaily">
       <property name="text">
        <string>概率</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_de">
         <property name="text">
          <string>德</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_cai">
         <property name="text">
          <string>才</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_gong">
         <property name="text">
          <string>功</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_zang">
         <property name="text">
          <string>脏</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_yun">
         <property name="text">
          <string>运</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_de">
         <property name="text">
          <string>25%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_cai">
         <property name="text">
          <string>25%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_gong">
         <property name="text">
          <string>25%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_zang">
         <property name="text">
          <string>25%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yun">
         <property name="text">
          <string>0%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_one">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>100</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 17), stop:0.65 rgba(159, 235, 148, 32), stop:0.721925 rgba(255, 238, 150, 65), stop:0.77 rgba(255, 128, 128, 102), stop:0.89 rgba(191, 128, 255, 32), stop:1 rgba(0, 0, 0, 0));
font: 20pt &quot;STKaiti&quot;;

</string>
    </property>
    <property name="text">
     <string>单抽</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="Label_shuxing">
       <property name="text">
        <string>属性</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_ming">
         <property name="text">
          <string>名</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_wen">
         <property name="text">
          <string>文</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_wu">
         <property name="text">
          <string>武</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_money">
         <property name="text">
          <string>财</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Label_ti">
         <property name="text">
          <string>体</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_ming">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_wen">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_wu">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_money">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_ti">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="card_one">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;STKaiti&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.5 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>德</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_two">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;STKaiti&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.2 rgba(255, 255, 255, 255), stop:0.8 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>【才】

干事利落
得官司务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_three">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;STKaiti&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.5 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>功</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_four">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;STKaiti&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.5 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>脏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_five">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;STKaiti&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.5 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="text">
     <string>运</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>510</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_yamen">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;户部&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_1">
         <property name="text">
          <string>尚书</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_1">
         <property name="text">
          <string>正一品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_2">
         <property name="text">
          <string>侍郎</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_2">
         <property name="text">
          <string>正二品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_3">
         <property name="text">
          <string>郎中</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_3">
         <property name="text">
          <string>正五品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_4">
         <property name="text">
          <string>员外</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_4">
         <property name="text">
          <string>从五品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_5">
         <property name="text">
          <string>主事</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_5">
         <property name="text">
          <string>正七品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_6">
         <property name="text">
          <string>司库</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_6">
         <property name="text">
          <string>正七品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_yamen_7">
         <property name="text">
          <string>司务</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_granjie_7">
         <property name="text">
          <string>从九品</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="history">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;STKaiti&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'STKaiti'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;【12岁】臣本布衣，躬耕于南阳。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;【18岁】因干事利落，被户部司务厅选中，得官司务。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>5</y>
      <width>811</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushButton_five</zorder>
   <zorder>label_suishu</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>pushButton_one</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>card_one</zorder>
   <zorder>card_two</zorder>
   <zorder>card_three</zorder>
   <zorder>card_four</zorder>
   <zorder>card_five</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>history</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
